pallets/phala/src/stakepool.rs

```diff
@@ -641,13 +672,21 @@ pub mod pallet {
 			rewards: BalanceOf<T>,
 		) {
 			if rewards > Zero::zero() {
-				if pool_info.total_shares == Zero::zero() {
+				if balance_close_to_zero(pool_info.total_shares) {
 					Self::deposit_event(Event::<T>::RewardDismissedNoShare(pool_info.pid, rewards));
 					return;
 				}
 				let commission = pool_info.payout_commission.unwrap_or_default() * rewards;
 				pool_info.owner_reward.saturating_accrue(commission);
-				pool_info.distribute_reward(rewards - commission);
+				let to_distribute = rewards - commission;
+				if is_positive_balance(to_distribute) {
```

Comment:
`is_positive_balance` is a little bit confused here, `to_distribute` is actually always a positive value, how about rename it to `not_dust_balance` or something else?

Reply:
Sounds good.